services/movie_session.py

```diff
@@ -0,0 +1,54 @@
+from typing import Optional
+
+from django.db.models import QuerySet
+
+from db.models import CinemaHall, Movie, MovieSession
+
+
+def create_movie_session(movie_show_time: str,
+                         movie_id: int,
+                         cinema_hall_id: int
+                         ) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall=CinemaHall(pk=cinema_hall_id)
+    )
+
+
+def get_movies_sessions(
+        session_date: Optional[str] = None
+) -> QuerySet:
+    movies_sessions = MovieSession.objects.all()
+    if session_date:
+        return MovieSession.objects.filter(
+            show_time__date=session_date
```

Comment:
Same here, you can do `movies = movies_sessions.filter(....)` and use one return in the end